Formula/spdlog.rb

```diff
@@ -56,7 +54,8 @@ def install
       }
     EOS
 
-    system ENV.cxx, "-std=c++11", "test.cpp", "-I#{include}", "-L#{Formula["fmt"].opt_lib}", "-lfmt", "-o", "test"
+    system ENV.cxx, "-std=c++11", "test.cpp", "-DSPDLOG_FMT_EXTERNAL", "-I#{include}", +
```

Comment:
The need to adjust the test this way makes me think removing the tweak might break things for users who don't use `pkg-config`. If this is the case, then that's a non-starter.

Reply:
fair enough, appreciate the prompt review

Reply:
One way to resolve this might be to modify the `spdlog.pc` file too so that it doesn't produce duplicate macro definitions. We'd need to check whether `tweakme.h` is always included, though.

That said, this seems like a lot of trouble to go through just to silence a warning.